lib/sycamore/sycamore/utils/cache_manager.py

```diff
@@ -1,11 +1,73 @@
 import hashlib
+import json
+import time
+from typing import Any
 
-from diskcache import Cache
+import diskcache
+from botocore.exceptions import ClientError
 
 
-class CacheManager:
+class Cache:
```

Comment:
Should we instead to CacheManager --> S3CacheManager(CacheManager) and DiskCacheManager(CacheManager) ?

Reply:
I wanted to rename CacheManager to Cache actually since there really isn't any management going on - it's just a cache interface. Thoughts on that?